src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -429,106 +419,96 @@ limitations under the License.
     </attribute>
-    <attribute code="0x0052" side="server" type="boolean" define="PRESETS_SCHEDULES_EDITABLE" writable="false" optional="true">PresetsSchedulesEditable</attribute>
-    <attribute code="0x0053" side="server" type="TemperatureSetpointHoldPolicyBitmap" define="TEMPERATURE_SETPOINT_HOLD_POLICY" default="0" writable="false" optional="true" min="0x0" max="0x3">TemperatureSetpointHoldPolicy</attribute>
-    <attribute code="0x0054" side="server" type="epoch_s" define="SETPOINT_HOLD_EXPIRY_TIMESTAMP" isNullable="true" writable="false" optional="true">SetpointHoldExpiryTimestamp</attribute>
-
+    <attribute code="0x0052" side="server" type="epoch_s" define="PRESETS_SCHEDULES_EDITABLE" optional="true" isNullable="true">SetpointHoldExpiryTimestamp</attribute>
     <!-- Client Commands -->
-
     <command source="client" code="0x00" name="SetpointRaiseLower" optional="false">
-      <description>
-        Command description for SetpointRaiseLower
-      </description>
-      <arg name="Mode" type="SetpointRaiseLowerModeEnum" min="0x0" max="0x2"/>
-      <arg name="Amount" type="int8s"/>
+      <description>Upon receipt, the attributes for the indicated setpoint(s) SHALL have the amount specified in the Amount field added to them.</description>
+      <arg id="0" name="Mode" type="SetpointRaiseLowerModeEnum" min="0x0" max="0x2"/>
+      <arg id="1" name="Amount" type="int8s"/>
     </command>
+
     <command source="client" code="0x01" name="SetWeeklySchedule" optional="true">
-      <description>
-        Command description for SetWeeklySchedule
-      </description>
+      <description>Upon receipt, the weekly schedule for updating setpoints SHALL be stored in the</description>
       <access op="invoke" privilege="manage"/>
-      <arg name="NumberOfTransitionsForSequence" type="int8u"/>
-      <arg name="DayOfWeekForSequence" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
-      <arg name="ModeForSequence" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
-      <arg name="Transitions" type="WeeklyScheduleTransitionStruct" array="true" length="10"/>
+      <arg id="0" name="NumberOfTransitionsForSequence" type="int8u"/>
+      <arg id="1" name="DayOfWeekForSequence" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
+      <arg id="2" name="ModeForSequence" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
+      <arg id="3" name="Transitions" type="WeeklyScheduleTransitionStruct" array="true" length="10"/>
     </command>
+
     <command source="client" code="0x02" name="GetWeeklySchedule" response="GetWeeklyScheduleResponse" optional="true">
-      <description>
-        Command description for GetWeeklySchedule
-      </description>
-      <arg name="DaysToReturn" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
-      <arg name="ModeToReturn" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
+      <description>Upon receipt, the unit SHOULD send in return the Get</description>
```

Comment:
Same comment as above ... gets cut in incomplete state

 follow up PR is ok

Reply:
This seems like a bug in Alchemy, since I am seeing it elsewhere too.  @hasty

Reply:
https://github.com/project-chip/alchemy/issues/4